src/lib.js

```diff
@@ -662,6 +670,26 @@ async function getItems(base64, customTextures = false, packs, cacheOnly = false
         }
       }
 
+      if (item.extra?.blocks_walked) {
+        let blocks_walked = item.extra.blocks_walked;
+
+        if (lore_raw) {
+          itemLore.push("", `§7Blocks Walked: §c${blocks_walked}`);
+          if (blocks_walked >= 100000) {
+            itemLore.push(`§8MAXED OUT!`);
+          } else {
+            let toNextLevel = 0;
+            for (const e of constants.prehistoric_egg_blocks_walked_ladder) {
+              if (blocks_walked < e) {
+                toNextLevel = e - blocks_walked;
+                break;
+              }
+            }
+            itemLore.push(`§8${toNextLevel} steps to tier up!`);
```

Comment:
"steps to tier up" sounds weird why don't we just put one line like "blocks walked 789/1000" and let people do the subtraction in their head

Reply:
Is "walk x blocks to tier up" better? I wanna keep the calculating to the computer.

Reply:
The more I think about it the more I think saying the remaining blocks is a good idea but "to tier up" still sounds kinda weird to me but I'm not sure what would be better